### ISSUE TITLE
Google Search url parameters are not changing after initial search

### ISSUE THREAD

**Issue by bspeare** [uiowa/uiowa#8321]
# Problem
Google Search url parameters are not changing after initial search so secondary search queries are not being tracked by Google Analytics.

# Steps to reproduce
Search at https://law.uiowa.edu/search?terms=levitt%20lecture and try searching again and see if the url parameter changes. 

# Proposed solution
- ?

**Comment by joewhitsitt** [comment]
not sure this is a bug. i don't think the code was originally was setup to do this. change to enhancement?

**Comment by mjoneill** [comment]
> not sure this is a bug. i don't think the code was originally was setup to do this. change to enhancement?

Agree. Not a bug, more like an unexpected gap based on how it was built. Fine with calling it a feature, enhancement, etc...

Seems to be working as developed. We just learned something about the boundaries of that.
